aspects/aspects.go

```diff
@@ -364,6 +364,18 @@ type Aspect struct {
 	bundle         *Bundle
 }
 
+type expandedMatch struct {
+	// path is dot-separated storage path without unfilled placeholders.
+	path string
```

Comment:
shouldn't this be called storagePath ? or it's because there are no plachedoler in it?  maybe then resolvedStoragePath?